src/extension.ts

```diff
@@ -34,7 +42,15 @@ export async function activate(context: ExtensionContext) {
     console.log(`VSpaceCode loaded: v${previousVersion} -> v${currentVersion}`);
     context.globalState.update(GlobalState.SpacecodeVersion, currentVersion);
     if (previousVersion === undefined) {
```

Comment:
It's kind of hard to clear the globalState from vscode. To test this, set this if condition to test.